jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -209,16 +209,17 @@
           api_hint="plg.aperture_method ="
           :api_hints_enabled="api_hints_enabled"
           :hint="'Extract '+resulting_product_name+' using an aperture masking method in place of the subset mask.'"
-        >
+        />
+        <v-row>
           <j-docs-link>
-            See the <j-external-link link='https://photutils.readthedocs.io/en/stable/aperture.html#aperture-and-pixel-overlap'
-            linktext='photutils docs'></j-external-link>
-            for more details on aperture masking methods.
+              See the <j-external-link link='https://photutils.readthedocs.io/en/stable/aperture.html#aperture-and-pixel-overlap'
+              linktext='photutils docs'></j-external-link> for more details on aperture masking methods.
           </j-docs-link>
-        </plugin-select>
```

Comment:
Huh, extra `</plugin-select>`? Is this a bug?

Reply:
nope, I replaced it above (line 212) with `/>` and moved the `j-docs-link` into its own row so that it looks better when the plugin is wide.